src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -145,6 +145,19 @@ private async Task CreateBlocksAsync()
                     if (((this.ibdState.IsInitialBlockDownload() || !this.connectionManager.ConnectedPeers.Any()) && !this.settings.BootstrappingMode)
                         || !this.federationManager.IsFederationMember)
                     {
+                        if (this.federationManager.IsFederationMember)
```

Comment:
It seems to me that this condition could be true with one of the other conditions in the enclosing `if` being the actual cause?

Reply:
Good spot! Fixed the typo.